spec/polymorphic_integer_type_spec.rb

```diff
@@ -117,6 +117,15 @@
     it "properly finds the object with a find_by" do
       expect(Link.find_by(source: source, id: link.id)).to eql link
     end
+
+    context "when source and target are namespaced without modifying polymorphic_name" do
```

Comment:
this spec was accidentally removed in an automatic merge conflict resolution, adding it back (was introduced [here](https://github.com/Drew-Goddyn/polymorphic_integer_type/pull/1)